src/components/main-content.tsx

```diff
@@ -10,16 +10,17 @@ import { OrganismsSpaceComponent } from "./spaces/organisms-space";
 interface IProps extends IBaseProps {}
 interface IState {}
 
-type SpaceComponent = typeof PopulationsSpaceComponent | typeof OrganismsSpaceComponent;
+type SpaceComponent = typeof PopulationsSpaceComponent |
+  typeof OrganismsSpaceComponent;
 
 type SpaceTypeToComponent = {
   [key in SpaceType]: SpaceComponent | undefined
 };
 
 const kSpaceComponents: SpaceTypeToComponent = {
   populations: PopulationsSpaceComponent,
-  breeding: undefined,
   organism: OrganismsSpaceComponent,
+  breeding: undefined,
```

Comment:
Just wondering, were the set of changes on this file requested by tslint? The current version doesn't seem to complain for me, on my editor. Don't know if we have different settings.

Reply:
I had a temporary space set up for testing the charts with the chart-test-space - this is a remnant of that testing and shall be removed, thanks!

Reply:
ah misunderstood - comment was referring to the change in the order of components (a remnant of testing) but has no effect on functionality.